controllers/contentlibrary/clustercontentlibraryitem/clustercontentlibraryitem_controller.go

```diff
@@ -111,9 +112,17 @@ func (r *Reconciler) ReconcileDelete(cclItem *imgregv1a1.ClusterContentLibraryIt
 		return nil
 	}
 
+	cvmi := &vmopv1.ClusterVirtualMachineImage{}
+	if err := r.Get(ctx, types.NamespacedName{Name: cvmiName}, cvmi); err != nil {
+		return client.IgnoreNotFound(err)
+	}
+
 	r.Metrics.DeleteMetrics(logger, cvmiName, "")
```

Comment:
Should this be moved inside finalizer check if block?

Reply:
Ditto.